ArduPlane/parachute.cpp

```diff
@@ -57,4 +58,38 @@ bool Plane::parachute_manual_release()
     return true;
 }
 
+/*
+    Parachute emergency check - trigger parachute when vertical speed threshold is reached
+    or a extreme low battery voltage threshold is reached.
+*/
+void Plane::parachute_emergency_check()
+{
+    // exit immediately if parachute or emergency mode is not enabled
+    if (!parachute.enabled() || !parachute.emergency_enabled() || parachute.released()) {
+        return;
+    }
+     
+    // do not release if vehicle is not flying
+    if (!is_flying()) {
+        return;
+    }
+     
+    // do not release if taking off 
+    if (auto_state.takeoff_complete == false ) {
+        return;
+    }
+    
+    // if the sink rate gets above the limits, release the parachute
+    if (auto_state.sink_rate > parachute.max_sink()) {
+        gcs().send_text(MAV_SEVERITY_CRITICAL,"Emergency Parachute: Stall detected");
+        parachute_release();
+    }
+    
+    // if the battery is almost dead, release parachute
+    else if (battery.voltage() < parachute.min_batt_voltage()) {
```

Comment:
The else needs to be after the previous brace, you can put the comment before the code below.

Reply:
Hello @OXINARF, i will fix this small issue and make my commits fits the rules.

Reply:
Should i rename the PR to Plane: Emergency Parachute Feature?